tests/unit/virtual-each-test.js

```diff
@@ -13,6 +13,7 @@ import hbs from 'htmlbars-inline-precompile';
 
 const { run } = Ember;
 const EXTRA_ROW_PADDING = 1;
+const SCROLL_TOP_BUFFER = 2;
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).